pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -490,6 +491,18 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 		return nil, errors.Wrap(err, "failed to determine if Octavia supports double listeners")
 	}
 
+	log.Print("Checking presence of Admission Controller daemonset")
+	if octaviaMultipleListenersSupport {
+		daemonset := &appsv1.DaemonSet{ObjectMeta: metav1.ObjectMeta{Namespace: KuryrNamespace, Name: names.KURYR_DNS_ADMISSION_CONTROLLER}}
```

Comment:
First and foremost shouldn't we remove the MutatingWebhookConfiguration? And also the Secrets?